src/test/java/racingcar/domain/RacingGameTest.java

```diff
@@ -18,38 +20,85 @@ private RacingCars createInitCars() {
-        RacingGame racingGame = new RacingGame(racingCars, 5);
+        final RacingCars racingCars = createInitCars();
+        final Round finalRound = new Round(5);
+        final RacingGame racingGame = new RacingGame(racingCars, finalRound);
         assertFalse(racingGame.isEnd());
     }
 
-    @ParameterizedTest
-    @ValueSource(ints = {0, -1})
-    @DisplayName("게임을 생성할 때 시도횟수가 0이하이면 예외가 발생한다.")
-    void nagativeCount(int count) {
-        RacingCars racingCars = createInitCars();
-        assertThatExceptionOfType(IllegalArgumentException.class)
-            .isThrownBy(() -> new RacingGame(racingCars, count))
-            .withMessageMatching("시도횟수는 0이하의 값이 들어올 수 없다.");
-    }
-
     @Test
     @DisplayName("게임이 종료되었는데 race할 경우 exception이 발생한다.")
     void raceEndException() {
-        RacingCars racingCars = createInitCars();
-        RacingGame racingGame = new RacingGame(racingCars, 1);
+        final RacingCars racingCars = createInitCars();
+        final Round finalRound = new Round(1);
+        final RacingGame racingGame = new RacingGame(racingCars, finalRound);
         racingGame.race();
-        assertThatExceptionOfType(RuntimeException.class)
+        assertThatExceptionOfType(IllegalStateException.class)
             .isThrownBy(() -> racingGame.race())
             .withMessageMatching("종료된 게임은 더이상 실행할 수 없다.");
     }
+
+    @Test
+    @DisplayName("단독 우승자가 있는 경우 우승자만 반환한다.")
+    void oneWinnerNames() {
+        // given
+        final List<RacingCar> racingCars = Arrays.asList(
+            new RacingCar(new Name("pobi"), () -> true),
+            new RacingCar(new Name("crong"), () -> false),
+            new RacingCar(new Name("honux"), () -> false)
+        );
+        final RacingGame racingGame = new RacingGame(new RacingCars(racingCars), new Round(1));
+
+        // when
+        racingGame.race();
+        final List<String> result = racingGame.winnerNames();
+
+        // then
+        assertAll(
+            () -> assertThat(result.size()).isEqualTo(1),
+            () -> assertThat(result).contains("pobi")
+        );
+    }
+
+    @Test
+    @DisplayName("우승자가 여러명인 경우 모두 반환한다.")
+    void multiWinnerNames() {
+        // given
+        final List<RacingCar> racingCars = Arrays.asList(
+            new RacingCar(new Name("pobi"), () -> true),
+            new RacingCar(new Name("crong"), () -> true),
+            new RacingCar(new Name("honux"), () -> false)
+        );
+        final RacingGame racingGame = new RacingGame(new RacingCars(racingCars), new Round(1));
+
+        // when
+        racingGame.race();
+        final List<String> result = racingGame.winnerNames();
+
+        // then
+        assertAll(
+            () -> assertThat(result.size()).isEqualTo(2),
+            () -> assertThat(result).containsExactly("pobi", "crong")
+        );
+    }
+
+    @Test
+    @DisplayName("종료되지 않은 게임은 우승자를 반환할 수 없다.")
+    void cannotGetWinnerInRunningGaem() {
```

Comment:
종료되지 않은 게임은 우승자를 반환할 수 있다는 로직이 추가되면서 테스트도 추가해주셨군요  👍 
```suggestion
    void cannotGetWinnerInRunningGame() {
```
오타가 하나 있네요 ㅋㅋ

Reply:
앗 ㅜㅜ 🤣